.github/workflows/ci.yml

```
@@ -0,0 +1,163 @@
+          cargo install cargo-outdated
+          cd whisper-rust
+          cargo outdated --exit-code 1
+
+      - name: Swift dependency resolution
+        run: swift package resolve
+
+      - name: Check Swift Package.resolved for issues
+        run: |
+          if [ -f Package.resolved ]; then
+            cat Package.resolved
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)